lib/hammer_cli_csv/base.rb

```diff
@@ -12,7 +12,7 @@ class BaseCommand < HammerCLI::Apipie::Command
     option %w(--threads), 'THREAD_COUNT', 'Number of threads to hammer with',
            :default => 1, :hidden => true
     option %w(--export), :flag, 'Export current data instead of importing'
-    option %w(--file), 'FILE_NAME', 'CSV file (default to /dev/stdout with --csv-export, otherwise required)'
+    option %w(--file), 'FILE_NAME', 'CSV file (default to /dev/stdout with --export, otherwise required)'
```

Comment:
This is changing the syntax used to run the command. Should we deprecate the old syntax or let user scripts just start breaking without warning?